src/test/java/teammates/test/cases/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -319,12 +322,6 @@ private void testSubmitAction() throws Exception {
         assertNull(BackDoor.getFeedbackResponse(fqPartial.getId(),
                                                 aliceTeam,
                                                 "Team 2"));
-        assertNull(BackDoor.getFeedbackResponse(fqConstSum2.getId(),
-                                                aliceTeam,
-                                                "Team 2"));
-        assertNull(BackDoor.getFeedbackResponse(fqConstSum2.getId(),
-                                                aliceTeam,
-                                                "Team 3"));
```

Comment:
Earlier in this test, we didn't fill any entries in `Q19` so we were verifying it with `assertNull`. But to test this new feature, we fill the `Q19` inputs with some value so I removed this.